com.unity.render-pipelines.universal/Runtime/ForwardRendererData.cs

```diff
@@ -62,6 +68,17 @@ public sealed class ShaderResources
             public Shader materialErrorPS;
         }
 
+        public ShaderResources shaders = null;
+
+        [Serializable, ReloadGroup]
+        public sealed class TextureResources
+        {
+            [SerializeField, Reload("Textures/numberFont.png")]
```

Comment:
TODO: We need to look at this font and what will be the design for light count debug. I remember this was very ugly/hacky from hack week.

Reply:
Still needs to be done - @andre-unity was going to take a look I think.

Reply:
@arttu-peltonen This could be something for the cross-pipeline team to look at - it would make sense for both render-pipelines to use the same font and approach to rendering it. Current "Debug.hlsl" has some code in it but URP doesn't use this (I don't know about HDRP) but even if it did I think it would look pretty ugly as it's just a set of bits, not an actual texture.

Reply:
@arttu-peltonen I think you've resolved this issue now haven't you? Can we just resolve this?

Reply:
Ah yes, this is fixed in my branch where I reused the tiled lighting debug mode from HDRP.